### PR TITLE
Add cluster up configuration for certificate signing duration.

### PR DESCRIPTION
**Pull request by jcbsmpsn**
```release-note
Add CLUSTER_SIGNING_DURATION environment variable to cluster configuration scripts
to allow configuration of signing duration of certificates issued via the Certificate
Signing Request API.
```

### CODE REVIEW

File: cluster/gce/config-test.sh [hunk 1]
```diff
@@ -367,6 +367,9 @@ ENABLE_PROMETHEUS_TO_SD="${ENABLE_PROMETHEUS_TO_SD:-true}"
 # Optional: Run kube-proxy as a DaemonSet if set to true, run as static pods otherwise.
 KUBE_PROXY_DAEMONSET="${KUBE_PROXY_DAEMONSET:-false}" # true, false
 
+# Optional: duration of cluster signed certificates.
+CLUSTER_SIGNING_DURATION="${CLUSTER_SIGNING_DURATION:-30m}"
```
Comment (yujuhong) [inline cluster/gce/config-test.sh]: This should not be enabled for all test clusters by default.

Comment (luxas) [reply cluster/gce/config-test.sh]: I'm curious, why not?

Comment (yujuhong) [reply cluster/gce/config-test.sh]: IIUC (which may not be the case), kubelet needs to rotate client certificate and it's not enabled by default.

Comment (luxas) [reply cluster/gce/config-test.sh]: So why do we then revert vs actually enabling kubelet cert rotation on kubelets?
cc @ericchiang 
(Not opposed, just asking)

Comment (ericchiang) [reply cluster/gce/config-test.sh]: @luxas I really just want to get the tests back to green as soon as possible. Once that's done happy to see other PRs modifying the test setup.

Comment (luxas) [reply cluster/gce/config-test.sh]: SGTM
@jcbsmpsn probably can create an issue to follow this up later

Comment (yujuhong) [reply cluster/gce/config-test.sh]: Yes, reverting is the standard practice in cases like this.

Also, I am unsure whether this should be tested in a separate job, or enabled by default for all test clusters.

**Pull request merged**